projects/plugins/boost/app/modules/cache/Boost_Cache.php

```diff
@@ -28,7 +23,7 @@ abstract class Boost_Cache {
 	public function __construct() {
 		$this->settings    = Boost_Cache_Settings::get_instance();
 		$this->request_uri = isset( $_SERVER['REQUEST_URI'] )
-			? $this->normalize_request_uri( $_SERVER['REQUEST_URI'] ) // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.ValidatedSanitizedInput.MissingUnslash
+			? Boost_Cache_Utils::sanitize_file_path( $this->normalize_request_uri( $_SERVER['REQUEST_URI'] ) ) // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.ValidatedSanitizedInput.MissingUnslash
```

Comment:
What kind of path should be passed to the `boost_cache_cacheable` hook in `is_cacheable`? Plain and unchanged? Normalized? Sanitized for filesystem storage? 

I feel like sanitizing for file-system storage then storing it as `$this->request_uri` in a base class that isn't aware of the filesystem storage is going to get confusing. :)